scripts/setup-database.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+distrofull=$(cat /etc/issue);
```

Comment:
See https://unix.stackexchange.com/a/84375
Let's have several  options in hand and test on all the distros we support. 

One more methos is here. 
```
cat /etc/*-release | grep "DISTRIB_ID=" | cut -d"=" -f2
```

Reply:
I have two distros on my system, works good on both. So maybe we can try going with this first :worried:

Reply:
`cat /etc/*-release | grep "DISTRIB_ID=" | cut -d"=" -f2` is better IMO. 
1. `${distrofull:: -6};` this might not always slice out all the escape characters
2. It's better to detect the base distro. Arch in this case. Which is correctly identified by the other command
![discheck](https://user-images.githubusercontent.com/6216000/34341989-3f4ad2ac-e9c9-11e7-95ea-176e9f150fe4.png)

Reply:
DONE